docs/src/components/Card.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React, { isValidElement } from 'react';
+import Box from '../../../src/Box/Box';
```

Comment:
@chrislloyd do we need the extra `components` directory? Could we use `docs/src` instead?

`../../..`

Reply:
Ah yes, this isn’t obvious in this PR - my next PR moves the docs into the src folder. This allows them to be organized in a DAG.